### PR TITLE
add esp32-c6 symbol

### PR DESCRIPTION
**Pull request by dreamcmi**
1. I created the symbols of QFN40 and QFN32 through the esp32-c6 datasheet on the espressif official website, and created the current pull request. Maybe the current symbol naming is not very reasonable., if you have better suggestions, please reply to me, I will fix it immediately.
2. The relevant design information of ESP32-C6-MINI-1 and ESP32-C6-WROOM-1 has not been published on the official website, so I did not submit this part of the content.
3. Added the description of ESP8684 symbol, because I forgot to add it in the last pull request.

### CODE REVIEW

**Comment by dreamcmi** [comment]
https://github.com/espressif/kicad-libraries/issues/69

**Comment by pedrominatel** [comment]
Hi, @dreamcmi

We are working on making all symbols and footprints compliant with the KLC.
Have you tried to run the symbols script checker?

https://klc.kicad.org/
https://gitlab.com/kicad/libraries/kicad-library-utils

Thank you so much for your contribution!

**Comment by dreamcmi** [comment]
> Hi, @dreamcmi
> 
> We are working on making all symbols and footprints compliant with the KLC. Have you tried to run the symbols script checker?
> 
> https://klc.kicad.org/ https://gitlab.com/kicad/libraries/kicad-library-utils
> 
> Thank you so much for your contribution!

I am correcting the symbol according to klc, but when running klc_check, the version of the symbol seems to be wrong. The version of this repo is 20211014, but the check script requires a version of 20220914. I temporarily modified this version number manually (it will not be pushed to the repo ), other work is in progress.
```shell
Traceback (most recent call last):
  File "./check_symbol.py", line 173, in check_library
    library = self._load_library(filename)
  File "./check_symbol.py", line 156, in _load_library
    return KicadLibrary.from_file(filename)
  File "/home/darren/kicad-check/kicad-library-utils/common/kicad_sym.py", line 1093, in from_file
    raise KicadFileFormatError(f'Version of symbol file is "{version}", not "20220914"')
kicad_sym.KicadFileFormatError: Version of symbol file is "20211014", not "20220914"
Could not parse library: ../../espressif-kicad-libraries/libraries/Espressif.kicad_sym. (Version of symbol file is "20211014", not "20220914")
Error: Version of symbol file is "20211014", not "20220914"
```

**Comment by dreamcmi** [comment]
I have corrected the symbols according to KLC, please review.
Although klc-check still says Violating S5.1, I don't know how to solve it now, and most of the existing symbols also have this problem.

File: libraries/Espressif.kicad_sym [hunk 1]
```diff
@@ -742,6 +742,364 @@
...
+      (pin power_in line (at -7.62 33.02 270) (length 2.54)
+        (name "VDDPST2" (effects (font (size 1.27 1.27))))
+        (number "28" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 -10.16 180) (length 2.54)
+        (name "U0TXD/GPIO16/FSPICS0" (effects (font (size 1.27 1.27))))
+        (number "29" (effects (font (size 1.27 1.27))))
+      )
+      (pin power_in line (at -12.7 33.02 270) (length 2.54)
+        (name "VDDA3P3" (effects (font (size 1.27 1.27))))
```
Comment (pedrominatel) [inline libraries/Espressif.kicad_sym:863]: Stack VDDA3P3 pins, like the GND.

**Review by pedrominatel — COMMENTED** [review]
Nitpicking on the symbol pin grouping.
I think we should try to meet the same pin grouping strategy as the other symbols.

My grouping suggestion:
image

**Comment by dreamcmi** [comment]
regrouped, please review

**Review by pedrominatel — APPROVED** [review]
LGTM

**Pull request merged**